### PR TITLE
Feature/11683 investigate housekeeping failure

### PR DESCRIPTION
**Pull request by mohamme61443**
raising PR for merge post successful testing/review

### CODE REVIEW

File: housekeeping/.vs/ProjectSettings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "CurrentProjectSetting": null
```
Comment (gavinwakeley1) [inline housekeeping/.vs/ProjectSettings.json:2]: Still seeing .vs content in the PR - can you remove this please via .gitignore?

File: housekeeping/.vs/VSWorkspaceState.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "ExpandedNodes": [],
```
Comment (gavinwakeley1) [inline housekeeping/.vs/VSWorkspaceState.json:2]: Still seeing .vs content in the PR - can you remove this please via .gitignore?

**Review by gavinwakeley1 — CHANGES_REQUESTED** [review]
Still seeing .vs content in the PR - can you remove this please via .gitignore?

**Comment by mohamme61443** [comment]
Looks like somebody else made changes to the branch hence despite me still ignoring them they are showing up

I never added those and they have been set to ignore on my client, also before doing a push I noticed them so decided to stash them before the push yet they are still showing. They're not my doing not sure where they came from.

KR
Ayaz

**Review by gavinwakeley1 — APPROVED** [review]
Looks good

**Pull request merged**